Shaders/FilmicAnamorphSharpen.fx

```diff
@@ -49,16 +49,23 @@ uniform float Offset <
 	ui_min = 0.0; ui_max = 2.0; ui_step = 0.01;
 > = 0.1;
 
-uniform int Contrast <
-	ui_label = "Edges mask";
+uniform bool DepthMask <
+	ui_label = "Enable edges masking";
+	ui_tooltip = "Depth high-pass mask switch";
+> = true;
+
+uniform int DepthMaskContrast <
```

Comment:
Note: Renaming this variable from `Contrast` to `DepthMaskContrast` will break existing presets using this effect. But I'll let you decide on whether you prefer backwards compatibility or code cleanness.

Reply:
I would go with old rule `Kaizen`; _Whenever there is an error don't hesitate to fix it even at cost, it will benefit the future_ ;)
So lets go with the code cleanness.